packages/mux-player/src/template.ts

```diff
@@ -28,6 +28,13 @@ const isLiveOrDVR = (props: MuxTemplateProps) =>
 
 export const content = (props: MuxTemplateProps) => html`
   <${unsafeStatic(castThemeName(props.theme) ?? 'media-theme-mux')}
+    src="${
+      !!props.src
+        ? props.src
+        : props.playbackId
+        ? getSrcFromPlaybackId(props.playbackId, { domain: props.customDomain, token: props.tokens.playback })
+        : false
+    }"
```

Comment:
I really think we shouldn't use `src` here as it's implying the value of src is a dependency for the the theme while it's not really. can this be renamed to something like `nosrc`, `unloaded`.

it's a bit a shame this has to be passed at all, is there any way this can be derived from within?

Reply:
Looks like we can do something like `hasSrc = this.querySelector('mux-video').hasAttribute('src')`. Does that make sense?

Reply:
Unfortunately, it looks like `hasAttribute()` on `mux-video` is too early.

Reply:
ok that's a shame. a shorter attribute makes sense then I think.

Reply:
Let me try and make it a boolean attribute like `has-src` or something